sdk/cosmos/azure-cosmos-spark_3_2-12/test-databricks/notebooks/basicScenarioAadCert.scala

```diff
@@ -0,0 +1,108 @@
+// Databricks notebook source
```

Comment:
I thought the Scala file names started with upper case?

Reply:
Not these notebooks - have been small letter from day 1 - rather keep it consistency within this folder.